config/development.json

```diff
@@ -28,8 +28,9 @@
 	"rebrand_cities_prefix": "rebrandcitiesdevelopmentsite",
 	"livechat_support_locales": [ "en", "es", "pt-br" ],
 	"features": {
-		"async-payments": false,
 		"ad-tracking": false,
+		"anchor-fm": false,
```

Comment:
Shouldn't we set this to `true` here so the feature is enabled while developing locally?